crates/milli/src/search/facet/search.rs

```diff
@@ -73,10 +76,16 @@ impl<'a> SearchForFacetValues<'a> {
         let index = self.search_query.index;
         let rtxn = self.search_query.rtxn;
 
-        let filterable_fields = index.filterable_fields(rtxn)?;
-        if !filterable_fields.contains(&self.facet) {
+        let filterable_attributes_rules = index.filterable_attributes_rules(rtxn)?;
+        if !is_field_facet_searchable(&self.facet, &filterable_attributes_rules) {
```

Comment:
check that facet search is enabled at all here?

Reply:
I am not sure to understand, do you speak about the following check?
https://github.com/meilisearch/meilisearch/blob/683a2ac685210402848466a6a0c9e8b406a4df52/crates/meilisearch/src/search/mod.rs#L1529-L1534

Reply:
ah, my bad, I didn't see this.